sdk/storage/azure-storage-file-share-stress/src/main/java/com/azure/storage/file/share/stress/App.java

```diff
@@ -18,8 +18,7 @@ public static void main(String[] args) {
             OpenInputStream.class,
             Upload.class,
             UploadFromFile.class,
-            UploadRange.class,
-            UploadRangeFromUrl.class
```

Comment:
Do we want to remove the `UploadRangeFromUrl.java` file as well?

Reply:
Thank you for catching this! Removed now